src/platform/common/configSettings.ts

```diff
@@ -90,6 +90,8 @@ export class JupyterSettings implements IWatchableJupyterSettings {
     // Hidden settings not surfaced in package.json
     public disableZMQSupport: boolean = false;
     // Hidden settings not surfaced in package.json
+    public useJupyterDebugger: boolean = false;
```

Comment:
Added a hidden setting that provides a way for us (jupyter engineers) to test the jupyter debugger even with local kernels.
Just like we test zmq.